chat/src/main/res/drawable/chat_mgs_edittext_border.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+    <selector xmlns:android="http://schemas.android.com/apk/res/android">
+        <item>
+            <shape android:shape="rectangle">
+                <solid android:color="#ffff"/>
+                <corners android:radius="10dp" />
+                <stroke
+                    android:width="2dp"
+                    android:color="#D63447"
```

Comment:
Its better if we don't use hardcoded colour like these, lets add it to a separate file and use its references